api/v1beta1/watcherapi_types.go

```diff
@@ -82,6 +82,13 @@ type APIOverrideSpec struct {
 type WatcherAPITemplate struct {
 	WatcherSubCrsTemplate `json:",inline"`
 
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=1
+	// +kubebuilder:validation:Maximum=32
+	// +kubebuilder:validation:Minimum=0
+	// Replicas of Watcher service to run
```

Comment:
WatcherAPI service to run

(I am duplicating this part of the change for the replicas in my wip for decisionengine until this merges)

Reply:
done